src/integration-test/tests/citizen/claimantResponse/pages/claimant-accept-payment-method.ts

```diff
@@ -0,0 +1,23 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  radioYes: 'input[id=acceptyes]',
+  radioNo: 'input[id=acceptNo]',
+  submit: 'input[id=saveAndContinue]'
```

Comment:
declare buttons outside of fields, as we do with other page objects
i.e 
`const buttons = {
      saveAndContinue: 'input[id=saveAndContinue]'
}`

Reply:
Done